catalog/Catalog.proto

```diff
@@ -81,13 +81,16 @@ message IndexScheme {
 }
 
 message CatalogRelationStatistics {
-  optional fixed64 num_tuples = 1;
+  optional bool is_exact = 1;
```

Comment:
I suggest to use a better name like `is_fresh`, as `is_exact` sounds more like a contrast to approximate stats, less like to indicate `up-to-date`.